Cargo.toml

```diff
@@ -95,6 +96,14 @@ syn = { version = "2.0.58", features = ["full"] }
 proc-macro2 = "1.0.79"
 quote = "1.0.35"
 
+# members
+openzeppelin-stylus = { path = "contracts" }
+openzeppelin-crypto = { path = "lib/crypto" }
+motsu = { path = "lib/motsu"}
+motsu-proc = { path = "lib/motsu-proc", version = "0.1.0" }
```

Comment:
Why do we set a specific version for `motsu-proc`? Is this intentional?

Reply:
Needed to publish `motsu` to crates.io; requirement of `cargo publish ...`